myget.bat

```diff
@@ -1,31 +1,24 @@
 @echo on
-call dnvm use 1.0.0-rc1-update1 -r clr
-IF ERRORLEVEL 1 goto error
-call dnu restore
-IF ERRORLEVEL 1 goto error
 
+dotnet restore || goto error
 
 set PROJECTS=src\Google.Api.CommonProtos src\Google.Api.Gax src\Google.Api.Gax.Rest testing\Google.Api.Gax.Testing
 set TESTS=test\Google.Api.Gax.Tests test\Google.Api.Gax.Rest.Tests
 
 for %%P in (%PROJECTS%) do (
   echo Building %%P
-  call dnu build --configuration %Configuration% %%P
-  IF ERRORLEVEL 1 goto error
+  dotnet build --configuration %Configuration% %%P || goto error
 )
 
 for %%P in (%TESTS%) do (
   echo Testing %%P
-  call dnu build --configuration %Configuration% %%P
-  IF ERRORLEVEL 1 goto error
-  dnx --configuration %Configuration% -p %%P test
-  IF ERRORLEVEL 1 goto error
+  dotnet build --configuration %Configuration% %%P || goto error
```

Comment:
True. I previously wanted to separate them to make it clear which bit had failed, but I'm happy to embrace a smaller build script.